packages/aws-appsync/src/link/signer/signer.ts

```diff
@@ -272,6 +272,9 @@ var sign = function (request, access_info, service_info = null) {
 *
 * @class Signer
 */
-export default class Signer {
+class Signer {
     static sign = sign;
 }
+
+export default Signer;
```

Comment:
Why you need both export? In which case you can use one or the other?

Reply:
It is just for convenience, so you can import it as:

```javascript
import Signer from 'foo/bar'
```

or

```javascript
import { Signer } from 'foo/bar'
```

Reply:
Cool thanks!